contracts/v0.1/scripts/generate-contract-addresses.js

```diff
@@ -1,10 +1,34 @@
 const path = require('path')
 const fs = require('fs')
+const axios = require('axios')
 const { loadJson, storeJson } = require('./utils.cjs')
 
 const ValidChains = ['baobab', 'cypress']
 const deploymentsPath = path.join(__dirname, '../deployments/')
 
+const getFeedTag = async (network, pairName) => {
+  url = `https://config.orakl.network/adapter/${network}/${pairName.toLowerCase()}.adapter.json`
```

Comment:
just optional comment, maybe you can use collected adapter json file and request url only once at the beginning

Reply:
I am not sure I understand. I think it should be called only once. It makes request only for `cypress` adapter.

Reply:
there's single json file collecting whole adapters together in following link

https://config.orakl.network/baobab_adapters.json

Reply:
Aha, now I get it! Good idea!

Reply:
fixed at https://github.com/Bisonai/orakl/pull/1364/commits/0bfc2a56a467fe5fd652ee69e5711c7081016567